packages/istanbul-reports/lib/html/index.js

```diff
@@ -94,7 +94,7 @@ const standardLinkMapper = {
     relativePath(source, target) {
         const targetPath = this.getPath(target);
         const sourcePath = path.dirname(this.getPath(source));
-        return path.posix.relative(sourcePath, targetPath);
+        return path.posix.relative(sourcePath, targetPath).replace(/index\.html$/, '');
```

Comment:
This change cannot be merged, many people (myself included) view the istanbul reports via filesystem.  So I'll run `nyc report --report html && firefox coverage/index.html`.  Removing the `index.html` breaks this report for viewers that are not viewing through a web server while also requiring that the web server be configured for `index.html` as directory index.

Reply:
I've made a new change because you are right, my change was not working in browser directly. So now the new approach is working in browser and server. Please review it.